build/dev-teardown.sh

```diff
@@ -5,6 +5,19 @@
 
 node_root=$(cd "$(dirname "$0")/.." && pwd)
 
+if [ -z "$1" ]; then
+  echo "Not removing any files because symlinks are in $node_root\npm-symlinks and teardown isn't necessary."
+  echo "If you want to tear down anyway, run dev-teardown.cmd with the --force flag"
+  exit 0
+fi
+
+if [ "$1" != "--force" ]; then
+  echo "usage: dev-teardown.cmd [--force]"
+  exit 1
+fi
+
+export NPM_CONFIG_PREFIX=$node_root/npm-symlinks
```

Comment:
I don't think this is necessary. we can remove the call to dev-teardown in the build definition.